manim/mobject/geometry.py

```diff
@@ -863,6 +863,119 @@ def __init__(self, n=6, **kwargs):
+    *arcs : Union[:class:`Arc`, :class:`ArcBetweenPoints`]
+        These are the arcs from which the arcpolygon is assembled.
+    **kwargs : Any
+        Affects how the ArcPolygon itself is drawn, but doesn't affect
+        passed arcs.
+
+    Attributes
+    ----------
+    arcs : :attr:`list`
+        The arcs used to initialize the ArcPolygon.
+
+    Examples
+    --------
+    One example of an arcpolygon is the Reuleaux triangle.
+    Instead of 3 straight lines connecting the outer points,
+    a Reuleaux triangle has 3 arcs connecting those points,
+    making a shape with constant width.
+
+    Passed arcs are stored as submobjects in the ArcPolygon.
+    This means that the arcs are changed along with the ArcPolygon,
+    for example when it's shifted, and these arcs can be manipulated
+    after the ArcPolygon has been initialized.
+
+    Also both the arcs contained in an ArcPolygon, as well as the
+    ArcPolygon itself are drawn, which affects draw time in ShowCreation
+    for example. In most cases the arcs themselves don't
+    need to be drawn, in which case they can be passed as invisible.
+        .. manim:: ArcPolygonExample
+
+            class ArcPolygonExample(Scene):
+                def construct(self):
+                    arc_conf = {"stroke_width":0}
+                    poly_conf = {"stroke_width":10,"stroke_color":BLUE,
+                      "fill_opacity":1,"color": PURPLE}
+                    a=[-1,0,0]
+                    b=[1,0,0]
+                    c=[0,np.sqrt(3),0]
+                    arc0=ArcBetweenPoints(a,b,radius=2,**arc_conf)
+                    arc1=ArcBetweenPoints(b,c,radius=2,**arc_conf)
+                    arc2=ArcBetweenPoints(c,a,radius=2,**arc_conf)
+                    reuleaux_tri=ArcPolygon(arc0,arc1,arc2,**poly_conf)
+                    self.play(FadeIn(reuleaux_tri))
+                    self.wait(2)
+
+    The ArcPolygon itself can also be hidden so that instead only the
+    contained arcs are drawn.
+        .. manim:: ArcPolygonExample2
+
+            class ArcPolygonExample2(Scene):
+                def construct(self):
+                    arc_conf = {"stroke_width":3,"stroke_color":RED,
+                      "fill_opacity":0.5,"color": GREEN}
+                    poly_conf = {"color":None}
+                    a=[-1,0,0]
+                    b=[1,0,0]
+                    c=[0,np.sqrt(3),0]
+                    arc0=ArcBetweenPoints(a,b,radius=2,**arc_conf)
+                    arc1=ArcBetweenPoints(b,c,radius=2,**arc_conf)
+                    arc2=ArcBetweenPoints(c,a,radius=2,**arc_conf)
+                    reuleaux_tri=ArcPolygon(arc0,arc1,arc2,**poly_conf)
+                    self.play(FadeIn(reuleaux_tri))
+                    self.wait(2)
+
+    ArcPolyon.arcs::
+        >>> from manim import ArcPolygon, Arc, ArcBetweenPoints
+        >>> ap=ArcPolygon(Arc(),ArcBetweenPoints([1,0,0],[0,1,0]),Arc())
+        >>> ap.arcs
+        [Arc, ArcBetweenPoints, Arc]
```

Comment:
There needs to be a blank line between the line ended by `::` and the following code block for it to render correctly. Also, is there a reason why the example is announced just by `ArcPolygon.arcs`? I'd like to see a sentence, similar to above.

Reply:
I was mostly just testing this to render and compute properly in the first place. I'll be reworking this again.